internal/command/controller/run.go

```diff
@@ -58,6 +59,8 @@ func newRunCommand() *cobra.Command {
 	_ = cmd.PersistentFlags().MarkHidden("experimental-rpc-v2")
 	cmd.PersistentFlags().BoolVar(&noExperimentalRPCV2, "no-experimental-rpc-v2", false,
 		"disable experimental RPC v2 (https://github.com/cirruslabs/orchard/issues/235)")
+	cmd.PersistentFlags().BoolVar(&enableTLS, "enableTLS", false,
+		"enable TLS")
```

Comment:
Let's be explicit that this is insecure and negate the flag to avoid confusion (people might think that they need to set `--enable-tls` explicitly to be secure):

```suggestion
	cmd.PersistentFlags().BoolVar(&noTLS, "insecure-no-tls", false,
		"disable TLS, making all connections to the controller unencrypted")
```

Reply:
Yes, this approach is safer.